src/components/profileImage/ProfileImage.js

```diff
@@ -0,0 +1,13 @@
+import ProfileCircle from "../../components/profileCircle";
+
+const ProfileImg = ({ profile }) => {
+  if (profile.profileImageUrl === "") {
+    return <ProfileCircle
+      initials={`${profile.firstName[0]} ${profile.lastName[0]}`}
+    />
+  }
+  else { return (<img className="profile-icon" src={profile.profileImageUrl} alt="profile Image"></img>); }
```

Comment:
Not sure if this was auto-formatted, but format this document, this line looks wierd